src/components/admin/rooms/Room.reducer.js

```diff
@@ -0,0 +1,80 @@
+import {
+    ADD_ROOM_REQUEST,
+    ADD_ROOM_SUCCESS,
+    ADD_ROOM_FAILURE,
+    GET_ROOM_REQUEST,
+    GET_ROOM_SUCCESS,
+    GET_ROOM_FAILURE,
+  } from "./Room.action.constants";
+  
+  const initialState = Object.freeze({
+    result: [],
+    requestStatus: null,
+    requestError: null,
+    getUserRes: [],
+    getUserStatus: null,
+    getUserError: null,
+  });
+  
+  export default function userReducer(state = initialState, action) {
```

Comment:
Update the function name

Reply:
[Delete issue fixed](https://github.com/vishwanath-somavarad/guest-house-portal/pull/2/commits/bb5700bf23d8b4dc294076a6f081828e273520fc)

already fixed in this commit